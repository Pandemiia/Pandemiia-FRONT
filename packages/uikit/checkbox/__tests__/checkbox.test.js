import React from 'react'
import { render } from 'enzyme'
import Checkbox from '../checkbox'

describe('uikit/checkbox', () => {
  it('should renders correctly', () => {
    const wrapper = render(<Checkbox value="some-val" checked />)
    expect(wrapper).toMatchSnapshot()
  })
})
