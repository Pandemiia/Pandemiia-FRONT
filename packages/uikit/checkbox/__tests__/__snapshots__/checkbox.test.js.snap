// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uikit/checkbox should renders correctly 1`] = `
<span
  class="dt-checkbox dt-checkbox-checked"
>
  <input
    checked=""
    class="dt-checkbox-input"
    type="checkbox"
  />
  <span
    class="dt-checkbox-inner"
  />
</span>
`;
